packages/ui-chart-components/src/utils/getChartTableData.tsx

```diff
@@ -5,29 +5,31 @@
 import React, { useMemo } from 'react';
 import styled from 'styled-components';
 import { formatDataValueByType } from '@tupaia/utils';
-import { DEFAULT_DATA_KEY, CHART_TYPES } from '../constants';
+import { BaseChartConfig, ValueType } from '@tupaia/types';
+import { DEFAULT_DATA_KEY } from '../constants';
+import { ChartType, LooseObject, TableAccessor } from '../types';
 import { formatTimestampForChart, getIsTimeSeries } from './utils';
 import { parseChartConfig } from './parseChartConfig';
+import { ViewContent } from '../types';
 
 // For the rowData, ignore labelType and use percentage instead of fractionAndPercentage as
 // we don't want to show multiple values a table cell
-const sanitizeValueType = valueType =>
-  valueType === 'fractionAndPercentage' ? 'percentage' : valueType;
-
-const getFormattedValue = (value, valueType) =>
+const sanitizeValueType = (valueType: ValueType): ValueType => {
+  return valueType === 'fractionAndPercentage' ? 'percentage' : valueType;
+};
+const getFormattedValue = (value: string | undefined, valueType: ValueType) =>
   value === undefined ? 'No Data' : formatDataValueByType({ value }, sanitizeValueType(valueType));
 
 const FirstColumnCell = styled.span`
   font-weight: 500;
   text-align: left;
 `;
 
-const makeFirstColumn = (header, accessor, sortRows) => {
-  const firstColumn = {
+const makeFirstColumn = (header: string, accessor: TableAccessor, sortRows?: Function) => {
+  const firstColumn: LooseObject = {
     Header: header,
     accessor,
-    // eslint-disable-next-line react/prop-types
-    Cell: ({ value }) => <FirstColumnCell>{String(value)}</FirstColumnCell>,
+    Cell: ({ value }: { value: string }) => <FirstColumnCell>{String(value)}</FirstColumnCell>,
```

Comment:
Is there any possibility that `value` isn't a `string`? Mainly because it gets converted to a string inside the cell above, so want to make sure it couldn't also be a number or something

Reply:
Yes good point. It's hard to know what the correct type should be. The value is table data and the String function defines the prop as any so I guess I'll go with any.